<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSelectivityMap</class>
 <widget class="QDialog" name="PlotSelectivityMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selectivity map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Select model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>358</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="modelBox">
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="plotterBox">
     <property name="minimumSize">
      <size>
       <width>551</width>
       <height>311</height>
      </size>
     </property>
     <property name="title">
      <string>Selectivity map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>932</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="closeButton">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>closeButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>plotterBox</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
